frontend/www/js/omegaup/components/arena/ContestPractice.vue

```diff
@@ -153,12 +153,8 @@ export default class ArenaContestPractice extends Vue {
     this.$emit('navigate-to-problem', request);
   }
 
-  onRunSubmitted(code: string, selectedLanguage: string): void {
-    const request = Object.assign({}, this.activeProblem, {
-      code,
-      selectedLanguage,
-    });
-    this.$emit('submit-run', request);
+  onRunSubmitted(request: { code: string; language: string }): void {
```

Comment:
```suggestion
  onRunSubmitted(run: { code: string; language: string }): void {
```

esto no es una petición: es un envío. en general "request" se debe limitar para los objetos que mandamos a través del API.

Reply:
Listo